frontend/javascripts/oxalis/model/sagas/volumetracing_saga.js

```diff
@@ -235,6 +256,172 @@ function* copySegmentationLayer(action: CopySegmentationLayerAction): Saga<void>
   }
 }
 
+const configureTensorFlow = (useWebworker, useGPU) => {
+  window.useWebworker = useWebworker;
+  window.useGPU = useGPU;
+  console.log("useWebworker set to", useWebworker, "and useGPU set to", useGPU);
+};
+
+// $FlowIgnore
+const isOffscreenCanvasSupported = typeof OffscreenCanvas !== "undefined";
+configureTensorFlow(isOffscreenCanvasSupported, true);
+window.configureTensorFlow = configureTensorFlow;
+
+const meanAndStdDevFromDataset = memoizeOne(
+  (dataset: APIDataset, layerName: string): Promise<{ mean: number, stdDev: number }> =>
+    getMeanAndStdDevFromDataset(dataset.dataStore.url, dataset, layerName),
+);
+
+function* inferSegmentInViewport(action: InferSegmentationInViewportAction): Saga<void> {
+  const allowUpdate = yield* select(state => state.tracing.restrictions.allowUpdate);
+  if (!allowUpdate) return;
+
+  const activeViewport = yield* select(state => state.viewModeData.plane.activeViewport);
+  if (activeViewport === "TDView") return;
+
+  const outputExtent = 100;
+  const inputContextExtent = 20;
+  const inputExtent = outputExtent + 2 * inputContextExtent;
+  const NUM_PREDICT_SLICES = 3;
+  const FLOODFILL_THRESHOLD = 0.7;
+
+  let aborted = false;
+  const toastConfig = {
+    onClose: () => {
+      aborted = true;
+    },
+    sticky: true,
+    key: "magic-wand",
+  };
+  const toastDescription = "Magic Wand is active. Close the toast to stop using it.";
+
+  Toast.info(toastDescription, toastConfig);
+
+  const colorLayers = yield* call([Model, Model.getColorLayers]);
+  // TODO: Which color layer to pick?
```

Comment:
Maybe show a warning if there is more than one color layers, which just says "Using color layer ${layerName} for automatic segmentation". That way, users can at least understand the limitation and let us know when we should add UI here.